synapse/python_dependencies.py

```diff
@@ -43,7 +43,6 @@
         "matrix_angular_sdk>=0.6.8": ["syweb>=0.6.8"],
     },
     "preview_url": {
-        "lxml>=3.6.0": ["lxml"],
```

Comment:
`lxml` doesn't come with a `__version__` fwiw.

Also, it seems to come as default in my python, (or at least its in the my system `dist-packages`)